src/model/block/IOpenResponseBlockConfig.ts

```diff
@@ -27,12 +27,6 @@ export interface IOpenResponseBlockConfig extends IBlockConfig {
      * @minimum 0
      */
     max_duration_seconds?: number
-  }
-
-  text?: {
-    /**
-     * @minimum 0
-     */
-    max_response_characters?: number
+    end_recording_digits?: string
```

Comment:
Yes, ✅ : This is correct.
Another way to confirm it is to click at the `proposed` link (status) we have in the proposal doc, and check the spec PR https://github.com/FLOIP/flow-spec/pull/64/commits/153673b76c139136b2bd7cd787fb674cfc5c8086#diff-34ca68ded23c9d851806c3bc5bf52394435cac0e4f9bfcabb587c350dc88b258R502